e2e/features/record-hearing-attendees-and-duration.feature

```diff
@@ -27,13 +27,68 @@ Feature: Record hearing attendees and duration
     And I generate the hearing bundle
     And I start decision and reasons
     And I generate decision and reasons
+
+  @regression @record-hearing-attendees @RIA-572
+  Scenario: Record hearing attendees and duration
+
     And I switch to be a `Admin Officer`
     And I click the `Overview` tab
     And I should see the `Overview` page
 
-  @regression @record-hearing-attendees @RIA-572
+    When I click the `Record attendees and duration` link
+    Then I am on the `Record attendees and duration` page
+
+    When I type `Judge Judy` for the `The judge` field
+    And I type `Frank` for the `The appellant (Optional)` field
+    And I type `Geoff` for the `The appellant's legal representative (Optional)` field
+    And I type `Sebastian` for the `The Home Office representative (Optional)` field
+    And I type `2` for the `Hours` field
+    And I type `30` for the `Minutes` field
+    And I click the `Continue` button
+    Then I am on the `Check your answers` page
+    And I should see `Judge Judy` for the `The judge` field
+    And I should see `Frank` for the `The appellant` field
+    And I should see `Geoff` for the `The appellant's legal representative` field
+    And I should see `Sebastian` for the `The Home Office representative` field
+    And I should see `2` for the `Hours` field
+    And I should see `30` for the `Minutes` field
+
+    When I click the `Save details` button
+    Then I should see the text `You have recorded the attendees and duration of the hearing`
+    And I should see the text `What happens next`
+    And I should see the text `The judge will record the decision and reasons.`
+    And I should see the text `You don't need to do anything more with this case.`
+
+    When I click the `Close and Return to case details` button
+    Then I should see an alert confirming the case `has been updated with event: Record attendees and duration`
+
+  @regression @RIA-1903
```

Comment:
We are duplicate the same line here may be we can add the scenario line asserting the attendees in the hearing Tab

Reply:
I have removed the duplicate scenario and updated the case-progression feature. It will check the Hearing attendance only for admin-officer role